styles.css

```diff
@@ -76,6 +74,10 @@ nav {
   padding: 10px 25px;
   border-radius: 0 0 0 70px;
   padding-top: 104px;
+
+  @media (min-width: 768px) {
+    background-image: url(images/Header\ bg@1x.png);
+  }
```

Comment:
- [ ] Your media query is not supposed to be inside a class-selector. Kindly remove it from it.

- [ ] It is required to use media query to make your desktop version of your portfolio. Since you follow mobile first approach, all the styling that would responsible for the desktop version should reside inside the media query syntax. It should look something similar to this. :point_down: 

```
   @media (min-width: 768px) {
        .headline {
           // code here.
         }
        // the rest of your code for the other sections on your 
            page.
    }
```
- [ ] You will continue all the styling for the desktop inside the media query. What the media query actual does is to take the width of your devise from 768px and above to display the way you want the content of the sections inside it to be.
- [ ] You must not forget that at the desktop version, your navbar items are visible but not on the mobile. Your hamburger icon is visible on the mobile version of your portfolio but not on the desktop. Kindly look into these and fix them.

- [ ] Your navbar and the headline sections on the desktop are supposed to look like this according to the Figma design. Kindly walk your design alongside the Figma design

### Your Design
![your-headline](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/26597432/20898f69-ad5a-4fe5-85c7-53858e1664ea)

### Figma Design
![temp-headline](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/26597432/c9fe3c2c-a403-4bd7-ace0-45eb66a8599c)

- [ ] You're required to stick to the Figma design as much as possible. Your design looks quite different from how it is supposed.  Kindly remove the horizontal scroll bar and ensure that there's equal left and right margin on the projects list. Kindly check the figma design and walk your design alongside the figma design

### Your Design
![your-project](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/26597432/c7792e63-a52c-483c-a003-bae3af1a2487)


### Figma Design
![temp-projects](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/26597432/ba3b6edd-6b9d-4053-b773-a62ac9d6cc0e)

- [ ] It is required to stick to the figma design as close as possbile. At this moment, your design for this section looks different to the figma design. Kindly check the screenshot below and walk your design alongside the figma design.

### Your Design
![your-about-me](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/26597432/70b05048-987c-49bb-8919-6a2467c5285f)

### Figma Design
![temp-about-me](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/26597432/9794317b-dbe3-4659-a2aa-1e707529a4ce)


- [ ] Kindly check your form section too and walk your design alongside the Figma design

### Your Design
![your-from](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/26597432/46e4f21f-e2ef-410b-9687-6ffc2fe1b1ad)

### Figma Design

![temp-form](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/26597432/81abeea6-ee18-40ba-b5ca-ccc84c0f6801)

**NOTE** You should not forget that styling your desktop version must not affect the mobile version you've worked on before. This could be achieved by putting all the styles responsible for the desktop version to be inside your media query syntax. I wish you the best of luck.